src/globus_cli/parsing/shared_options/__init__.py

```diff
@@ -313,6 +313,7 @@ def decorate(f: C) -> C:
             f = click.option(
                 "--anonymous",
                 "principal",
+                type=click.Tuple([str, str]),
```

Comment:
Note: these are necessary as without them, the value gets injected as a string rather than a tuple:
```python
principal="('anonymous', '')"
```